setup.cfg

```diff
@@ -6,3 +6,7 @@ omit= *__init__*, *__version__*
 show_missing=True
 precision=2
 sort=Miss
+
+[options.entry_points]
+console_scripts =
+	rvl = nervaluate.cli:app
```

Comment:
This is the problem. `rvl` should be `nervaluate`. It's the entry point to the script, and short for `reviews-labeler`

Reply:
ok thank you. I just fixed it.